tests/release/e2e-test-releaseplan-owner-ref.go

```diff
@@ -0,0 +1,59 @@
+package release
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
+)
+
+var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-added", Label("release", "releaseplan-ownerref", "HACBS"), func() {
+	defer GinkgoRecover()
+	var fw *framework.Framework
+	var err error
+	var devNamespace string
+	var releasePlan *releaseApi.ReleasePlan
+
+	BeforeAll(func() {
+		fw, err = framework.NewFramework("releaseplan-ownerref")
+		Expect(err).NotTo(HaveOccurred())
+		devNamespace = fw.UserNamespace
+
+		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, "managed", "")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = fw.AsKubeAdmin.HasController.CreateApplication(applicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	AfterAll(func() {
+		if !CurrentSpecReport().Failed() {
+			Expect(fw.SandboxController.DeleteUserSignup(fw.UserName)).To(BeTrue())
+		}
+	})
+
+	var _ = Describe("ReleasePlan verification", Ordered, func() {
+
+		It("verifies that the ReleasePlan has an owner reference for the application", func() {
+			releasePlan, err = fw.AsKubeAdmin.ReleaseController.GetReleasePlan(sourceReleasePlanName, devNamespace)
```

Comment:
should we check for `err != nil` on the line below (i.e. `Expect(err).ShouldNot(HaveOccurred())`)

Reply:
fixed